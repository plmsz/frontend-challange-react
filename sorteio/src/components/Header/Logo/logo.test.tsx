import { Logo } from './index'
import { render, screen } from '@testing-library/react'

it('renders correctly', () => {
  const { asFragment } = render(<Logo title='MEGA-SENA' />)

  const title = screen.getByRole('heading', { name: /mega-sena/i })
  expect(title).toBeInTheDocument()
  expect(asFragment()).toMatchSnapshot('create')
})
