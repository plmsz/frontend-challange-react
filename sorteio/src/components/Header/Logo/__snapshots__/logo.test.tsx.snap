// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly: create 1`] = `
<DocumentFragment>
  <div
    class="sc-bczRLJ dZSZMH"
  >
    <picture>
      <source
        media="(max-width: 768px)"
        srcset="logoPrimary.svg"
      />
      <source
        media="(min-width: 768px)"
        srcset="logoSecondary.svg"
      />
      <img
        alt="logo - trevo de 4 folhas"
        src="logoPrimary.svg"
      />
    </picture>
    <h1>
      MEGA-SENA
    </h1>
  </div>
</DocumentFragment>
`;
